src/context/product-context.js

```diff
@@ -0,0 +1,26 @@
+// import { createContext, useContext, useEffect, useState } from "react";
```

Comment:
try to avoid pushing commented code in PR

Reply:
Removed! Pushed that by mistake, thanks for informing! ✨